www/install.rst

```diff
@@ -26,10 +25,55 @@ which includes all the key packages:
 * `Pyzo <http://www.pyzo.org/>`_: A free distribution based on Anaconda and the
   IEP interactive development environment.  Supports Linux, Windows and Mac.
 
-Linux packages
---------------
+Installing via pip
+------------------
+
+Mac and Linux users can install pre-built binary packages for the scipy stack
+using `pip <https://pip.pypa.io/en/stable>`_.  pip is the installation tool
+recommended by the Python Packaging Authority.
+
+At the moment, *pip does not work well for Windows* because we do not have the
+necessary compilers to build some binary packages on Windows.
+
+To install via pip on Mac or Linux, first upgrade pip to the latest version.
+For Python 3::
+
+    python3 -m pip install --upgrade pip
+
+Replace ``python3`` with ``python2`` for Python 2 pip upgrade.
+
+Then install the scipy stack packages with pip.  We recommend a *user*
```

Comment:
A couple of people just went through the NumPy code base and capitalized everything, so I feel obliged to say ``SciPy Stack`` please:)